roles/create-jira-project/tasks/create_project.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Create Sample Project
+  uri:
+    url: "{{ atlassian_url }}/rest/api/2/project"
+    method: POST
+    user: "{{ atlassian_username }}"
+    password: "{{ atlassian_password }}"
+    return_content: yes
+    force_basic_auth: yes
+    body_format: json
+    header:
+      - Accept: 'application/json'
+      - Content-Type: 'application/json'
+    body: "{ 'key': '{{ atlassion_key }}',
+             'name': '{{ atlassion_project }}',
+             'projectTypeKey': 'software',
```

Comment:
Please make these inventory variables, but defaulted to these values - i.e.: 
```
'projectTypeKey': "{{ atlassian.jira.projectTypeKey | default('software') }}"
```

Same for the others below with "hardcoded" values.